app/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,8 @@
     <string name="error_webhook_human_not_found">Human not found</string>
 
     <string name="pip_phone_enter_split">Enter split screen</string>
+    <string name="shortcut_splitscreen_short">Split screen</string>
+    <string name="shortcut_splitscreen_long">Start split screen with game</string>
```

Comment:
@kamieniarz Would you help translate the new strings please? 🥺

Reply:
Sorry, I haven't noticed you mentioned me

    <string name="shortcut_splitscreen_short">Podzielony ekran</string>
    <string name="shortcut_splitscreen_long">Podziel ekran z grą</string>